src/NuGet.Services.AzureSearch/SearchService/SearchParametersBuilder.cs

```diff
@@ -34,6 +34,8 @@ public class SearchParametersBuilder : ISearchParametersBuilder
         private static readonly List<string> SortableTitleDesc = new List<string> { IndexFields.SortableTitle + Desc, IndexFields.Created + Desc }; // Last title by lex order first, then oldest
         private static readonly List<string> CreatedAsc = new List<string> { IndexFields.Created + Asc }; // Newest first
         private static readonly List<string> CreatedDesc = new List<string> { IndexFields.Created + Desc }; // Oldest first
+        private static readonly List<string> TotalDownloadsAsc = new List<string> { IndexFields.Search.TotalDownloadCount + Asc }; // Least downloads first
```

Comment:
Consider sorting by `Created` as a second level of sorting so that you have more stability. A similar approach was taking on the other sort-by. We did this because Azure Search has undefined order when the a sort-by is tied (internally I think it is by document ID which is nondeterministic and can vary from query to query due to sharding)

Reply:
If you add this, you'll need to update the in-service sorting as well